.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
What is this file? maybe you need to add it to .gitignore

Reply:
done